conans/cli/cli.py

```diff
@@ -126,6 +132,14 @@ def run(self, *args):
             self._print_similar(command_argument)
             raise ConanException("Unknown command %s" % str(exc))
 
+        if (
+            command != "config"
+            or (
+                command == "config" and len(args[0]) > 1 and args[0][1] != "install"
```

Comment:
I think auto-config-install might be skipped if you are executing a ``config`` command, because for example you might want to launch a manual config yourself, or remove the config origins or something.